src/libs/actions/Session.js

```diff
@@ -124,46 +177,7 @@ function signIn(password, twoFactorAuthCode) {
         email: credentials.login,
     })
         .then((authenticateResponse) => {
-            const autoGeneratedLogin = Str.guid('expensify.cash-');
-            const autoGeneratedPassword = Str.guid();
-
-            API.CreateLogin({
-                authToken: authenticateResponse.authToken,
-                partnerName: CONFIG.EXPENSIFY.PARTNER_NAME,
-                partnerPassword: CONFIG.EXPENSIFY.PARTNER_PASSWORD,
-                partnerUserID: autoGeneratedLogin,
-                partnerUserSecret: autoGeneratedPassword,
-                doNotRetry: true,
-                email: authenticateResponse.email,
-            })
-                .then((createLoginResponse) => {
-                    if (createLoginResponse.jsonCode !== 200) {
-                        throw new Error(createLoginResponse.message);
-                    }
-
-                    setSuccessfulSignInData(createLoginResponse);
-
-                    // If we have an old generated login for some reason
-                    // we should delete it before storing the new details
-                    if (credentials.autoGeneratedLogin) {
-                        API.DeleteLogin({
-                            authToken: authenticateResponse.authToken,
-                            partnerUserID: credentials.autoGeneratedLogin,
-                            partnerName: CONFIG.EXPENSIFY.PARTNER_NAME,
-                            partnerPassword: CONFIG.EXPENSIFY.PARTNER_PASSWORD,
-                            doNotRetry: true,
-                        })
-                            .catch(console.debug);
-                    }
-
-                    Onyx.merge(ONYXKEYS.CREDENTIALS, {autoGeneratedLogin, autoGeneratedPassword});
-                })
-                .catch((error) => {
-                    Onyx.merge(ONYXKEYS.ACCOUNT, {error: error.message});
-                })
-                .finally(() => {
-                    Onyx.merge(ONYXKEYS.ACCOUNT, {loading: false});
-                });
+            createTemporaryLogin(authenticateResponse);
```

Comment:
NAB, if we only need the `authToken` and `email` maybe we can pass these are separate parameters to `createTemporaryLogin()`.

Reply:
NAB? 
I'll fix that :)

Reply:
Ah sorry this is shorthand for "not a blocker" but will make someone happy if you decide to fix